draft-ietf-taps-interface.md

```diff
@@ -1740,6 +1755,36 @@ per-Message basis using the Transmission Profile Message Property; see
 {{send-profile}}.
 
 
+### Bounds on Send or Receive Rate
+
+This property specifies an upper-bound rate that a transfer is not expected to
+exceed (even if flow- and congestion control allow higher rates), and/or a
```

Comment:
Having `flow-` and `congestion control` seems unbalanced. Either:
- `flow- and congestion-control`
- `flow control and congestion control`